hamocc/bodensed.F90

```diff
@@ -79,33 +82,60 @@ subroutine bodensed(kpie,kpje,kpke,pddpo)
      write(io_stdo_bgc,*)  ' '
   endif
 
-  porwat(1) = 0.85
-  porwat(2) = 0.83
-  porwat(3) = 0.8
-  porwat(4) = 0.79
-  porwat(5) = 0.77
-  porwat(6) = 0.75
-  porwat(7) = 0.73
-  porwat(8) = 0.7
-  porwat(9) = 0.68
-  porwat(10) = 0.66
-  porwat(11) = 0.64
-  porwat(12) = 0.62
+  ! this initialization can be done later via reading a porosity map
+  if (l_sed_por)then
+   ! lon-lat variable sediment porosity from input file
+   call apply_sedpor(kpie,kpje,ks,omask,sed_por)
```

Comment:
I find it a bit overkill to define an extra module for ~10 lines of code (essentially porwat=sedpor) that could be just placed directly here.

Reply:
:-) - agree - I was debating with myself - I'll change it.